src/types/declaration.d.ts

```diff
@@ -7,3 +7,12 @@ declare module "*.css" {
   const content: { [className: string]: string };
   export default content;
 }
+
+declare global {
```

Comment:
Could you create an issue to handle this better ? Looks like latest version of web3 export the types

Reply:
created #97